packages/react-components/react-calendar-compat/stories/Calendar/CalendarCustomDayCellRef.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Calendar } from '@fluentui/react-calendar-compat';
+import type { CalendarDayProps } from '@fluentui/react-calendar-compat';
+
+export const CalendarCustomDayCellRef: React.FunctionComponent = () => {
+  const [selectedDate, setSelectedDate] = React.useState<Date>();
+
+  const onSelectDate = React.useCallback((date: Date, _): void => {
+    setSelectedDate(date);
+  }, []);
+
+  const customDayCellRef: CalendarDayProps['customDayCellRef'] = React.useCallback((element, date, classNames) => {
```

Comment:
customDayCellRef is a bit of an odd name for a callback as it isn't a ref.

Reply:
sadly I can't change this for now since we are trying to keep the API backwards compatible. I think the name comes from it kind of behaving like a callback ref.